### PR TITLE
Add s_ol/0xc_pad default layout

### PR DESCRIPTION
**Pull request by s-ol**
## Description

This is the default layout for the 0xC.pad macropad.
See also the keyboard PR qmk/qmk_firmware#16057.

### CODE REVIEW

**Comment by s-ol** [comment]
@noroadsleft qmk/qmk_firmware#16057 was merged, so this could be tested now 🙂

File: public/keymaps/s/s_ol_0xc_pad_default.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  "keyboard": "s_ol/0xc_pad",
+  "keymap": "default",
+  "commit": "102185a40214d975054d83d6ca74613373e64f64",
```
Comment (noroadsleft) [inline public/keymaps/s/s_ol_0xc_pad_default.json:4]:
```suggestion
  "commit": "8cc86490aac59a93ece62229cc93097a17420fa5",
```

**Review by noroadsleft — CHANGES_REQUESTED** [review]

**Review by noroadsleft — APPROVED** [review]

**Comment by noroadsleft** [comment]
Thanks!

**Comment by s-ol** [comment]
Thank *you*! 🙂

**Pull request merged**